libs/databrickscfg/profiles.go

```diff
@@ -99,3 +100,11 @@ func LoadProfiles(path string, fn ProfileMatchFunction) (file string, profiles P
 
 	return
 }
+
+func ProfileCompletion(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	_, profiles, err := LoadProfiles(DefaultPath, func(p Profile) bool { return true })
```

Comment:
What you suggest should be easy. You can traverse `cmd` all the way to root and figure out of any of the parts is `account`. If so, we complete account profiles, otherwise workspace profiles.

Reply:
I see. I think to make this actually work, we need to do a slightly more subtle change:
1. If no command group is set at all, complete with both profile types.
2. If the command group is "accounts", complete with account profiles only.
3. Otherwise, complete with workspace profiles only.

Reply:
Otherwise, `databricks --profile <TAB>` would only show workspace profiles.

Reply:
Just tried it, and `args` is always empty even if there are arguments supplied. I guess the values supplied to `args` are only in the context of the command that the flag is defined on... I just refactored this function for now. In the future, we can autogenerate the --profile flag in command groups.